assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -16,16 +18,66 @@ import {
 } from '../../../actions/adminActions/landingPage';
 import { createRandomId } from '../../../utils/globalFunctions';
 
+export type LandingPageModule = {
+  defaultOrder: number,
+  editableOrder: boolean,
+  id: string,
+  identifier: string,
+  moduleId: string,
+  required: boolean,
+  title: string
+};
+
 type Props = {
   enabledModules: List<Map>,
-  moduleTypes: Array<Object>,
+  moduleTypes: Array<LandingPageModule>,
   locale: string,
   modulesById: Map<string, Map>,
   moveModuleDown: Function,
   moveModuleUp: Function,
   toggleModule: Function,
   createModule: Function
 };
+
+const MODULES_IDENTIFIERS = {
+  header: 'HEADER',
+  timeline: 'TIMELINE',
+  chatbot: 'CHATBOT',
+  topThematics: 'TOP_THEMATICS',
+  tweets: 'TWEETS',
+  contact: 'CONTACT',
+  data: 'DATA',
+  news: 'NEWS',
+  partners: 'PARTNERS',
+  introduction: 'INTRODUCTION',
+  footer: 'FOOTER'
+};
+
+export const addCountSuffix = (modules: Array<LandingPageModule>): Array<LandingPageModule> => {
+  // This function counts the occurencies of each module titles
```

Comment:
Your comment is misleading, I prefer something like:

`// Add a suffix to the title of the module if this module appears more than one time. This suffix is the number of times this module appeared in the array.`

Maybe we can change the function name to: `addEnumSuffixToModuleTitles`